runtime/commonTest/src/kotlinx/serialization/json/serializers/JsonArraySerializerTest.kt

```diff
@@ -40,6 +36,47 @@ class JsonArraySerializerTest : JsonTestBase() {
         assertEquals(string, prebuiltJson.toString())
     }
 
+    @Test
+    fun testMissingCommas() = parametrizedTest { useStreaming ->
+        val message = "Expected end of the array or comma"
+        assertFailsWith<JsonDecodingException>(message) { nonStrict.parse(JsonArraySerializer, "[a b c]", useStreaming) }
```

Comment:
Well, this too can be factored out (probably with usage of local functions) like

```kotlin
fun doTest(input: String) = assertFailsWith<JsonDecodingException>(message) { nonStrict.parse(JsonArraySerializer, input, useStreaming) }
doTest(...)
```